ArduPlane/quadplane.h

```diff
@@ -205,6 +207,7 @@ class QuadPlane
     AC_PosControl *pos_control;
     AC_WPNav *wp_nav;
     AC_Loiter *loiter_nav;
+    AP_LandingDetector *landing_detector;
```

Comment:
This is never filled in. The others here are created by quadplane. The other way is the same as AHRS where you take a reference and init it in the quadplane constructor.